packages/frontend/components/Button/index.tsx

```diff
@@ -0,0 +1,14 @@
+import classnames from 'classnames';
+
+function Button ({size, bgColor, textColor, children}) {
+    return (
+      <button className={classnames(`bg-${bgColor} text-${textColor} py-2 px-4 rounded-lg hover:bg-${bgColor}-dark active:bg-${bgColor}-darker`, {
```

Comment:
This is great, just one thing did you try to build? sometimes postcss purge get confused with template strings, you can learn more about it here:
https://tailwindcss.com/docs/optimizing-for-production